### PR TITLE
Fixes #11010 - removed type and association tabs.

### PR DESCRIPTION
**Pull request by alongoldboim**
Placed those options in the template tab, removed the headers file because those tabs are no longer needed.

### CODE REVIEW

File: app/views/templates/_form.html.erb [hunk 1]
```diff
@@ -4,8 +4,7 @@
   <%= base_errors_for @template %>
   <ul class="nav nav-tabs" data-tabs="tabs">
     <li class="active"><a id="primary_tab" href="#primary" data-toggle="tab"><%= _("Template") %></a></li>
-
-    <%= render "#{@type_name_plural}/custom_tab_headers" %>
+    <%= render "#{@type_name_plural}/custom_tab_headers" if type == 'provisioning_template'%>
```
Comment (ares) [inline app/views/templates/_form.html.erb]: I don't think we can simply limit this to provisioning_templates, plugins may add their own custom tabs.

Comment (alongoldboim) [inline app/views/templates/_form.html.erb]: you are right. ill do it the other way.

File: app/views/ptables/_custom_tabs.html.erb [hunk 2]
```diff
@@ -1,11 +1,7 @@
-<div class="tab-pane" id="template_type">
   <%= checkbox_f f, :snippet, :onchange => "snippet_changed(this)", :label=>_('Snippet'), :disabled => @template.locked? %>
```
Comment (dLobatog) [inline app/views/ptables/_custom_tabs.html.erb]: Can you move `Snippet` and `Operating system` family below `Default`? They are critical fields yet they are hidden under `Audit comment` and the upload Template button, which are much less important.

**Comment by dLobatog** [comment]
@alongoldboim :+1: quick and clean, thank you. If you could just move the fields up let's :shipit: after that.

**Comment by alongoldboim** [comment]
@eLobato I think were good :).

**Comment by dLobatog** [comment]
Merged as b39b95bfef4cbdf3cb74a5dc5a50fb50516e0c67, thanks @alongoldboim!

**Pull request closed without merging**